src/System.Diagnostics.EventLog/tests/EventLogEntryCollectionTests.cs

```diff
@@ -40,7 +40,7 @@ public void CopyingEventLogEntryCollection()
             finally
             {
                 EventLog.DeleteEventSource(source);
-                Helpers.RetryOnWin7(() => EventLog.Delete(log));
+                Helpers.RetryOnAllPlatforms(() => EventLog.Delete(log));
```

Comment:
Does anything still use the RetryOnWin7 helper, or can it be deleted?

And why is the "OnAllPlatforms" needed? Can't it just be "Retry"? Can it use our shared helper instead?

Reply:
Good feedback, yes the differentiation isn't needed anymore. I collapsed the helper methods together. The shared helper is already used underneath, it just sets some defaults.